zk_prover/README.md

```diff
@@ -62,22 +62,28 @@ Furthermore the benchmarking function `verify_zk_proof_benchmark` will also prin
 
 Run on MacBook Pro 2023, M2 Pro, 32GB RAM, 12 cores
 
+Benches run after PR #80 (`add solidity verifier`). In order to achieve small proof size, to be cheap to verify on-chain. We decreased the RANGE_BITS in which the balances should lie to 64 and reduced the rate of the Poseidon hash function to 1. 
```

Comment:
You mention this same pull request here, and it already has the benches - is that correct?

Reply:
Yes, these benches are run based on this branch. Is that ok?

Also, I am aware that the process of benchmarking should be more structured. Do you have any experience with that?

Reply:
Yes, these benches are run based on this branch. Is that ok?

Also, I am aware that the process of benchmarking should be more structured. Do you have any experience with that?

Reply:
I mean do you really want to leave a reference to some PR in the docs?